src/app/Main/Body/QueryPage/Services/ContainerServices/initializeData.service.tsx

```diff
@@ -0,0 +1,60 @@
+import { ApiResponse, callApiService } from "../../../../../Core/callApi.service";
+import { IsolateCountedDTO } from "../../../../../Shared/Model/Api_Isolate.model";
+import { FILTER_URL, ISOLATE_COUNT_URL } from "../../../../../Shared/URLs";
+import {
+    FilterInterface,
+    ClientFiltersConfig,
+    ClientSingleFilterConfig,
+} from "../../../../../Shared/Model/Filter.model";
+import { adaptFilterFromApiService } from "../adaptFilterFromAPI.service";
+import { generateUniqueValuesService } from "../generateUniqueValues.service";
+import { FilterConfigDTO } from "../../../../../Shared/Model/Api_Filter.model";
+
+export async function initializeDataService(): Promise<{

```

Comment:
Same here: fetchInitialDataService would be better

Reply:
Done.